pages/character/ranking.go

```diff
@@ -2,7 +2,6 @@ package character
 
 import (
 	"net/http"
```

Comment:
As mentioned in the [style guidelines](https://github.com/akyoto/quality/blob/master/STYLE.md#use-goimports) the `goimports` command should be run after saving the file. It will automatically reorder the imports and add a newline between stdlib and other imports.